scripts/link-checker.js

```diff
@@ -142,7 +142,18 @@ async function checkLinks() {
               .querySelectorAll('[id]')
               .map((idElement) => idElement.getAttribute('id'));
 
-            return { ok: response.ok, status: response.status, ids };
+            // Handle GitHub README links.
+            // These links are stored within a react-partial element
+            const ssr = html
+              .querySelectorAll('react-partial')
+              .filter(
+                (partialElement) =>
+                  partialElement.getAttribute('partial-name') ===
+                  'repos-overview' // This is the partial that handles the READMEs
+              )
+              .flatMap((element) => element.getElementsByTagName('script'))
+              .map((element) => JSON.parse(element.innerHTML))[0];
+            return { ok: response.ok, status: response.status, ids, ssr };
```

Comment:
Would it be possible to add an option for this work in the `.link-checker` file? If we can avoid dumping this complexity in the main script that would be a big win.